packages/esm-patient-medications-app/src/order-basket/drug-search.ts

```diff
@@ -20,12 +19,21 @@ import { Drug } from '../types/order';
 //
 // This method certainly isn't perfect, but again, since the common medication data is only available to us, it's kind of
 // the best thing we can do here.
+interface daysDurationUnit {
+  uuid: string;
+  display: string;
+}
 
-export async function searchMedications(searchTerm: string, encounterUuid: string, abortController: AbortController) {
+export async function searchMedications(
+  searchTerm: string,
+  encounterUuid: string,
+  abortController: AbortController,
+  configDaysDurationUnit: daysDurationUnit,
```

Comment:
Remove the `config` prefix; it doesn't matter to this function where the value comes from.